### ISSUE TITLE
what MSE actually does?

### ISSUE THREAD

**Issue by iperov** [natanielruiz/deep-head-pose#62]
part of loss is mse ( predicted * idx_tensor, yaw )
```
idx_tensor = [idx for idx in xrange(66)]
idx_tensor = Variable(torch.FloatTensor(idx_tensor)).cuda(gpu)

yaw_predicted = torch.sum(yaw_predicted * idx_tensor, 1)
loss_reg_yaw = reg_criterion(yaw_predicted, label_yaw_cont)
```

but `torch.sum(yaw_predicted * idx_tensor, 1)` has infinite solutions?
for example
for target angle 45
bins 44 = 0.5 , 46 = 0.5 are correct answer.

so what point to use such approach?

**Comment by imdeepmind** [comment]
Check #46
